packages/api/src/api/users.ts

```diff
@@ -0,0 +1,20 @@
+import { Router } from 'express';
+import { User } from '../entities/User';
+
+export const users = Router();
+
+users.get('', async (_req, res) => {
+
+  const userRepository = _req.entityManager.getRepository(User);
+  const selectedUser = await userRepository.findAll(); 
+
+  res.send({selectedUser});
+});
+
+users.get('/:userId', async (req, res) => {
+
+  const userRepository = req.entityManager.getRepository(User);
+  const selectedUser = await userRepository.findOne({ id : req.params.userId }); 
+
+  res.send({selectedUser});
+});
```

Comment:
- When you are using a parameter, make sure to remove the `_` in front of it. That's only there when it's not being used.
- You don't have to get a repository before finding stuff. You can use the `entityManager` directly
- The return type of `/users` should be an array of users, so you can just send that along
- The return type of `/users/:userId` should be a single user, we don't need to wrap it in another object.

```suggestion
users.get('', async (req, res) => {
  const users = await req.entityManager.find(User);

  res.send(users);
});

users.get('/:userId', async (req, res) => {
  const user = await req.entityManager.findOne(User, { id : req.params.userId }); 

  res.send(user);
});
```